src/OrbitQt/CallTreeViewItemModel.cpp

```diff
@@ -161,10 +203,23 @@ QVariant CallTreeViewItemModel::GetForegroundRoleData(const QModelIndex& index)
   ORBIT_CHECK(index.isValid());
   auto* item = static_cast<CallTreeNode*>(index.internalPointer());
   auto* unwind_errors_item = dynamic_cast<CallTreeUnwindErrors*>(item);
-  if (unwind_errors_item != nullptr && index.column() == kThreadOrFunction) {
-    static const QColor kUnwindErrorsColor{Qt::red};
+  auto* unwind_error_item = dynamic_cast<CallTreeUnwindErrorNode*>(item);
+
+  if ((unwind_errors_item != nullptr || unwind_error_item != nullptr) &&
+      index.column() == kThreadOrFunction) {
+    static const QColor kUnwindErrorsColor{QColor::fromRgb(255, 128, 0)};
     return kUnwindErrorsColor;
   }
+
+  const auto* parent_is_unwind_errors_item =
+      dynamic_cast<const CallTreeUnwindErrors*>(item->parent());
+  const auto* parent_is_unwind_error_item =
+      dynamic_cast<const CallTreeUnwindErrorNode*>(item->parent());
+  if ((parent_is_unwind_error_item != nullptr || parent_is_unwind_errors_item != nullptr) &&
+      index.column() == kThreadOrFunction) {
+    static const QColor kUnwindErrorFunctionColor{QColor::fromRgb(150, 150, 150)};
```

Comment:
Can we make this slightly brighter? At least from the screenshot it looked hard to read. Maybe `Qt::lightGray`.